test/test_nn.py

```diff
@@ -4470,6 +4470,37 @@ def test_load_state_dict_ref_cycle(self):
 
         self.assertEqual(refcycles, 0)
 
+    def test_load_state_dict_custom(self):
+
+        class CustomState(nn.Module):
+            def __init__(self):
+                super(CustomState, self).__init__()
+                self.param = torch.nn.Parameter(torch.ones(1))
+                self.sub = torch.nn.Linear(5,5)
+
+            def _save_to_state_dict(self, destination, prefix, keep_vars):
+                destination[prefix + "serialized"] = self.param.data + 1
+
+            def _load_from_state_dict(self, state_dict, prefix, local_metadata,
```

Comment:
Do we really want to be overriding this method directly?

Reply:
It seems to be symmetric to the _save_to_state_dict (mostly, except hooks). Better suggestions?